source/templates/add_step_sidebar.slim

```diff
@@ -39,5 +39,11 @@
 					== svg("workflow/icon-add_step-show_all")
 					span == data[:strings][:workflows][:steps][:add][:show_all_steps]
 			ol.steps[ng-if="!addStepCtrl.isCategoryMode"]
+				div[ng-if="addStepCtrl.shouldFilterByProjectType && addStepCtrl.filteredSteps(category.steps).length == 0"]
+					p We didn't find any results for your search in {{ appService.appDetails.projectTypeID }} steps category, would you like to search all steps?
```

Comment:
I think you should create a string map for the project types. I'm not sure what's the whole list but for instance:

- ios -> iOS
- android -> Android
- react (or what it's called) -> React Native
... defaulting to the type id if not specified

Another option is to omit the category name from the message if this is too complicated.